src/browser_launchers/playwright_launcher.js

```diff
@@ -64,15 +64,40 @@ export class PlaywrightLauncher extends BrowserLauncher {
             launcher = BrowserLauncher.requireLauncherOrThrow('playwright', 'apify/actor-node-playwright-*').chromium,
         } = launchContext;
 
+        const { launchOptions = {}, ...rest } = launchContext;
+
         super({
-            ...launchContext,
+            ...rest,
+            launchOptions: {
+                ...launchOptions,
+                executablePath: getDefaultExecutablePath(launchOptions),
```

Comment:
How does this work with the existing `useChrome` option ?

Reply:
Good point. This way, it would never use chrome in playwright-chrome image 👍